### PR TITLE
Neume module guidelines v4 update

### PR DESCRIPTION
**Pull request by vigliensoni**
This pull request (PR) updates the MEI Neume module guidelines to MEI4.0. It provides:

- An overview of the Neumes module with a description of the basic elements of the module in MEI4.0
- Background information about the Neumes module
- A section about the challenges of encoding neume notation and how the new MEI4.0 Neumes module tackles these issues
- Samples of MEI encoding in different notations

Additional comments about the PR:

- In order to preserve the V3.0 guidelines, none of the old MEI examples or images were deleted. New MEI examples and images were added for V4.0
- Some images in this PR were added using an `` element instead of using plain Markdown. This was necessary in order to have control of the width attribute of those images.
- A “live” version of the pull request can be accessed at https://ddmal.music.mcgill.ca/guidelines/v4/content/neumes.html

This update to the guidelines was prepared by Elsa De Luca (@elsinhadl), Ichiro Fujinaga (@fujinaga), and Gabriel Vigliensoni (@vigliensoni).

### CODE REVIEW

**Comment by kepper** [comment]
Dear Neumes IG, thanks for this wonderful PR. We especially love that you've set up a site where we can see this live. We still need to read through everything (even though we don't expect any problems), but one question: Have you double-checked all examples to be valid 4.0, or, if they're not, indicated so? If that is the case already, there seems little work left before it can be merged. Thanks again!

**Comment by vigliensoni** [comment]
Dear MEI Technical Chairs,

The examples were reviewed with care but we did not validate them. This is not indicated in the guidelines. I guess the best solution would be to double-check they are valid 4.0. Is this something you may want to do or would you prefer the IG to take care of this?

**Comment by kepper** [comment]
If you had the ressources to do that, it would be highly appreciated ;-)

**Comment by vigliensoni** [comment]
Perfect, we will do it. If there any updates, I will add more commits to this PR. 

Thank you

**Comment by kepper** [comment]
I thank you :-)

**Comment by vigliensoni** [comment]
Dear @kepper 

We double-checked and validated all the MEI Neume module code examples so that all of them now conform MEI4.0 We also the updated the documentation accordingly.

Let us know if you need anything else from us, thanks.

This update to the guidelines was prepared by Elsa De Luca (@elsinhadl), Ichiro Fujinaga (@fujinaga), and Gabriel Vigliensoni (@vigliensoni).

**Comment by kepper** [comment]
Thanks for the brilliant work. I hope to find some time early next week to deal with this…

File: _includes/v4/examples/neumes/neumes-sample170.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-            <note oct="3" pname="g"></note>
-         </uneume>
-      </rdg>
-      <rdg source="R"></rdg>
+   <rdg source="#D">
+   <uneume name="punctum" xml:id="neumes.d19e">
+   <note oct="3" pname="g"></note>
+   </uneume>
+   </rdg>
+   <rdg source="R"></rdg>
```
Comment (bwbohl) [inline _includes/v4/examples/neumes/neumes-sample170.xml]: inconsistent use of `@source`

Comment (bwbohl) [reply _includes/v4/examples/neumes/neumes-sample170.xml]: as datatype is data.URI value should be '#R'
please check for all other cases

**Comment by vigliensoni** [comment]
Hi @bwbohl, since the two examples you sent have `<uneume>` tags, I think those are legacy files in the repository.

**Comment by bwbohl** [comment]
> Hi @bwbohl, since the two examples you sent have `<uneume>` tags, I think those are legacy files in the repository.

Are those files still needed for some examples in the documentation? Else you could delete them from the v4/examples

**Comment by vigliensoni** [comment]
@bwbohl The neume example folder is now clean from legacy examples.

**Comment by kepper** [comment]
Dear @vigliensoni, as this properly documents current v4, I would like to pull this in now (finally… apologies for being so incredibly late). The version warning is still left, though. Do you prefer to remove that by yourself, or shall I remove it after merging? If I don't hear back to you within the next day or so, I'll pull and take care of it by myself…

**Pull request merged**